src/chainstate/stacks/miner.rs

```diff
@@ -3994,6 +4006,131 @@ pub mod test {
         }
     }
 
+    #[test]
+    fn test_build_anchored_blocks_bad_nonces() {
+        let mut privks = vec![];
+        let mut balances = vec![];
+        let num_blocks = 10;
```

Comment:
Because this test only tests against 4 tenures, `num_blocks` should be 4.

Reply:
I had used 10 as an arbitrary number to show that the log output kept on showing for each new block, but can reduce to 4.